realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -462,6 +477,15 @@ private static String getManagementRealmUrl(URL authUrl) {
         }
     }
 
+    /**
+     * FIXME Javadoc
+     *
+     * @return
+     */
+    public PermissionManager getPermissionManager() {
```

Comment:
I don't think there is any way around requiring a looper thread for now. Waiting for specific events from the server is going to be really painful otherwise. At least for a v1 I think it is acceptable.